### ISSUE TITLE
master has a timeshift error

### ISSUE THREAD

**Issue by bpbond** [JGCRI/gcamdata#438]
```
Error in enc2utf8(col_names(col_labels, sep = sep)) : 
  argumemt is not a character vector
```

**Comment by rplzzz** [comment]
Don't forget, you can use `git bisect` to track down the commit that introduced the error.

**Comment by rplzzz** [comment]
Also, are you sure this is an error in the time shifting test?  It looks to me like the latest build failed before it could even install the package, let alone run the tests.

**Comment by bpbond** [comment]
Resolved in #456
